src/Runner/Hook.php

```diff
@@ -283,18 +310,9 @@ private function executeFailAfterAllActions(array $actions): void
      */
     private function handleAction(Config\Action $action): void
     {
-        if ($this->shouldSkipActions()) {
-            $this->io->write(
-                $this->formatActionOutput($action->getAction()) . ': <comment>deactivated</comment>',
-                true
-            );
-            return;
-        }
-
         $this->io->write(' - <fg=blue>' . $this->formatActionOutput($action->getAction()) . '</> : ', false);
 
-        if (!$this->doConditionsApply($action->getConditions())) {
-            $this->io->write('<comment>skipped</comment>', true);
+        if ($this->checkSkipAction($action)) {
```

Comment:
I did this to reduce the amount of code in the `handleAction()` method and, hopefully, to simplify the logic, so it's easier to read and understand.

Reply:
👍